### PR TITLE
Fine edits to doc and omiserver.conf before final release

### PR DESCRIPTION
**Pull request by yakman2020**
@Microsoft/omi-devs

### CODE REVIEW

File: Unix/installbuilder/conf/omiserver.conf [hunk 1]
```diff
@@ -20,6 +20,18 @@ httpsport=0
...
+## of the credentials file. This file is in the form
+##
+## <domain>|<username>:<password> 
+##
+## For further information, please see link: https://github.com/Microsoft/omi/blob/master/Unix/doc/setup-ntlm-omi.md
+##
+##
+##NtlmCredsFile=PATH
+## For example
+NtlmCredsFile=/etc/opt/omi/.creds/ntlm
```
Comment (paulcallen) [inline Unix/installbuilder/conf/omiserver.conf]: you are checking in the actual cred file location? I think this is just an example that is not commented out

Comment (jeffaco) [inline Unix/installbuilder/conf/omiserver.conf]: Oh, good catch. For what's checked in, this should be commented out (right Paul?).

The customer should enable this only if they wanted to.

I'd also get rid of the example, change to something like:

```
## To enable NTLM authentication via negotiation (SPNEGO), specify the location 
## of the credentials file. This file is in the form
##
## <domain>|<username>:<password> 
##
## For further information, please see link: https://github.com/Microsoft/omi/blob/master/Unix/doc/setup-ntlm-omi.md
##
#NtlmCredsFile=/etc/opt/omi/.creds/ntlm
```

**Comment by yakman2020** [comment]
done

File: Unix/installbuilder/conf/omiserver.conf [hunk 2]
```diff
@@ -20,6 +20,18 @@ httpsport=0
...
+## of the credentials file. This file is in the form
+##
+## <domain>|<username>:<password> 
+##
+## For further information, please see link: https://github.com/Microsoft/omi/blob/master/Unix/doc/setup-ntlm-omi.md
+##
+##
+##NtlmCredsFile=PATH
+## For example
+#NtlmCredsFile=/etc/opt/omi/.creds/ntlm
```
Comment (jeffaco) [inline Unix/installbuilder/conf/omiserver.conf]: I think you missed my prior comment. Please delete lines 30-32, you don't need them.

**Pull request merged**